Resources/translations/pagerfanta.sw.xliff

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="sw" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>previous</source>
+                <target>Nyumba</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>next</source>
+                <target>Mbele</target>
+            </trans-unit>
+        </body>
+    </file>
+</xliff>
```

Comment:
Please could you add a linebreak here to [keep git happy](https://stackoverflow.com/q/5813311/328817)?

Reply:
Just added the line-break,I hope its all good